File: dev/requirements/screen/cases/card-states.4.2.case.swift
#if canImport(SwiftUI) && os(iOS)
import SetlistUI

/// 4.2 — told, live and queued side by side in one grid.
let cardStates_4_2 = RequirementCase(id: "4.2") {
    try MainActor.assumeIsolated {
        try expectScreen(StageScreen(state: StageFixture.state()), slug: "card-states", id: "4.2", region: .grid)
    }
}
#endif

File: dev/requirements/screen/cases/card-states.4.2.case.dart
import 'package:setlist_ui/setlist_ui.dart';

import '../../shared/requirement_case.dart';
import '../../shared/snapshot.dart';

/// 4.2 — Told, live and queued cards are each coloured differently.
final cardStates_4_2 = RequirementCase('4.2', (tester) async {
  await expectRegion(
    tester,
    StageFixture.state(),
    region: StageRegion.grid,
    slug: 'card-states',
    id: '4.2',
  );
});
